experimental/packages/opentelemetry-instrumentation-grpc/package.json

```diff
@@ -18,7 +18,8 @@
     "watch": "tsc --build --watch",
     "precompile": "lerna run version --scope $(npm pkg get name) --include-dependencies",
     "prewatch": "node ../../../scripts/version-update.js",
-    "peer-api-check": "node ../../../scripts/peer-api-check.js"
+    "peer-api-check": "node ../../../scripts/peer-api-check.js",
+    "postinstall": "node -e \"console.log(\\\"@opentelemetry/instrumentation-grpc - warning: The package 'grpc' (https://www.npmjs.com/package/grpc) is deprecated. It will no longer be instrumented in the next release of '@opentelemetry/instrumentation-grpc'. Please migrate to '@grpc/grpc-js' (https://www.npmjs.com/package/@grpc/grpc-js) to continue receiving telemetry.\\\");\""
```

Comment:
Message looks good, my only thought is adding color to make it pop more for those who need to see it.
 
Is it excessive to have a file `postinstall.js` that includes this message e.g. `"postinstall": "node postinstall.js"` and then

```js
// postinstall.js
console.log('\x1b[95m%s\x1b[0m', '-----------------------------------------');
console.log('\x1b[95m%s\x1b[0m', '|        WARNING: Important Notice       |');
console.log('\x1b[95m%s\x1b[0m', '-----------------------------------------');
console.log(
  '\x1b[95m%s\x1b[0m',
  '"\\ @opentelemetry/instrumentation-grpc - warning: The package `grpc` (https://www.npmjs.com/package/grpc) is deprecated.'
);
console.log(
  '\x1b[95m%s\x1b[0m',
  'It will no longer be instrumented in the next release of `@opentelemetry/instrumentation-grpc`.'
);
console.log(
  '\x1b[95m%s\x1b[0m',
  'Please migrate to `@grpc/grpc-js` (https://www.npmjs.com/package/@grpc/grpc-js) to continue receiving telemetry.'
);
console.log('\x1b[95m%s\x1b[0m', '-----------------------------------------');
```

![postinstall-warning](https://github.com/open-telemetry/opentelemetry-js/assets/29520003/f62d990e-8679-4228-bf79-908a72aecece)

Reply:
I can go either way on colors/separate file

Reply:
Eh I think it's good as-is, shipit :shipit:

Reply:
I like the idea, let's add the color, but don't do the banner. This way, we can keep the script inline in `package.json` but it still makes it pop a bit more :slightly_smiling_face: 

Done in [18ad5e5](https://github.com/open-telemetry/opentelemetry-js/pull/3833/commits/18ad5e5a960e0e7230d3556b42acef15a73c2c00)